forge-gui/res/cardsfolder/s/sparas_bodyguard.txt

```diff
@@ -0,0 +1,15 @@
+Name:Spara's Bodyguard
+ManaCost: G W U
+Types: Creature Rhino Warrior
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigChoose | TriggerDescription$ When CARDNAME enters the battlefield, you may choose a creature card in your hand. If you do, it perpetually gains “This creature enters the battlefield with an additional shield counter on it.” Otherwise, put a shield counter on CARDNAME.
+SVar:TrigChoose:DB$ ChooseCard | ChoiceZone$ Hand | Choices$ Creature.YouOwn | ChoiceTitle$ You may choose a creature card in your hand | RememberChosen$ True | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | StaticAbilities$ PerpetualShield | Name$ Spara's Bodyguard's Perpetual Effect | Duration$ Permanent
+SVar:PerpetualShield:Mode$ Continuous | Affected$ Card.ChosenCard | AddKeyword$ etbCounter:SHIELD:1 | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ The chosen card perpetually enters the battlefield with an additional shield counter.
+SVar:DBAddCounter:DB$ PutCounter | Defined$ Self | ConditionDefined$ ChosenCard | ConditionNotPresent$ Card | CounterType$ SHIELD | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearChosenCard$ True
+T:Mode$ Phase | Phase$ BeginCombat | TriggerZones$ Battlefield | Execute$ TrigPump 
+SVar:TrigPump:DB$ Pump | Defined$ Self | NumAtt$ +X | NumDef$ +X | TriggerDescription$ At the beginning of each combat, Spara's Bodyguard gets +1/+1 until end of turn for each shield counter among other creatures you control.
```

Comment:
TriggerDescription$ needs to be moved to the trigger line
Spara's Bodyguard = CARDNAME